xc7/utils/prjxray_tile_import.py

```diff
@@ -275,22 +278,44 @@ def import_tile(db, args):
     for the entire tile if args.fused_sites is set to true.
     """
 
-    tile = db.get_tile_type(args.tile)
+    import_tiles = []
+    if args.import_tiles:
+        import_tiles = set(args.import_tiles.split(','))
+    else:
+        import_tiles = {args.tile}
+
+    sites = reduce(lambda acc, tile: acc + list(db.get_tile_type(tile).get_sites()), import_tiles, [])
 
     # Wires sink to a site within the tile are input wires.
     input_wires = set()
 
     # Wires source from a site within the tile are output wires.
     output_wires = set()
 
+    inner_pins = {
```

Comment:
Please add a comment on what this is, and what it should become in the future.  Please add an issue and have the comment link to the issue.